services.py

```diff
@@ -0,0 +1,212 @@
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+import numpy as np
+
+
+class Data_parsing:
+    def __init__(self):
+        self.headers = {
+            'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3'
+        }
+
+    @staticmethod
+    def __reading_the_number_of_pages(content) -> int:
```

Comment:
Why static method is private ?

Reply:
Because it cannot be used anywhere except for the method that is in this class